app/src/main/java/org/simple/clinic/facility/change/FacilityChangeScreen.kt

```diff
@@ -6,6 +6,7 @@ import android.util.AttributeSet
 import android.view.View
```

Comment:
@sanchita the first commit copies a feature from `RegistrationFacilitySelectionScreen` to `FacilityChangeScreen` for showing the search field only when the facilities are available and shown on screen.